jest.conf.json

```diff
@@ -2,6 +2,7 @@
   "preset": "./node_modules/carbon-factory/jest.conf.json",
   "notify": false,
   "setupFiles": [
+    "raf/polyfill",
```

Comment:
I wonder if this can be added to carbon-factory instead? Then everyone should get it